api/app/Models/PoolCandidate.php

```diff
@@ -205,7 +208,12 @@ public function scopePoolCandidateStatuses(Builder $query, ?array $poolCandidate
 
     public static function scopeAvailable(Builder $query): Builder
     {
-        return $query->whereIn('pool_candidate_status', [ApiEnums::CANDIDATE_STATUS_QUALIFIED_AVAILABLE, ApiEnums::CANDIDATE_STATUS_PLACED_CASUAL]);
+        $query->whereIn('pool_candidate_status', [ApiEnums::CANDIDATE_STATUS_QUALIFIED_AVAILABLE, ApiEnums::CANDIDATE_STATUS_PLACED_CASUAL])
+            ->where(function ($query) {
```

Comment:
Thanks for wrapping this in an extra `where`! :star:  I'm a little worried about the comparison to date("Y-m-d") since I think it will allow records that were suspended on the current date to get pulled up.  I suggest a comparison to Carbon::now or use [an expression](https://laravel.com/docs/9.x/queries#raw-expressions) to compare to the database `current_timestamp`.

Reply:
Updated to use Carbon::now with [dbe6843](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5832/commits/dbe6843b78b9d53dc832f61752dc6a1957d6ac80).